extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -261,17 +212,14 @@ public Description getDescription() throws ParsingException {
     }
 
     @Override
-    public int getAgeLimit() throws ParsingException {
-        assertPageFetched();
-        if (!isAgeRestricted) {
+    public int getAgeLimit() {
```

Comment:
The result of this function should, too, being cached, since it is used many times